cyences_app_for_splunk/default/data/ui/views/cs_splunk_admin_reports.xml

```diff
@@ -16,15 +16,27 @@
     <latest></latest>
     <sampleRatio>1</sampleRatio>
   </search>
+  <search id="base_search">
+    <query>| rest splunk_server=* /services/server/status/partitions-space 
```

Comment:
Will the rest-handler will show the status of all the instances?

I would using index=introspection would cover all the full instances including HFs, IDXs, and other SHs.
Also, we will be able to calculate Disk, Memory, and CPU from a single query.

That way we can complete CY-50 and CY-51 together with just one search too.

Reply:
Rest will provide all the Splunk instances which include Search Heads, Indexers, CM, DS but not HFs. I am changing the search to use _introspection index.